clippy_lints/src/utils/paths.rs

```diff
@@ -80,6 +81,7 @@ pub const RANGE_TO_INCLUSIVE: [&str; 3] = ["core", "ops", "RangeToInclusive"];
 pub const RANGE_TO_INCLUSIVE_STD: [&str; 3] = ["std", "ops", "RangeToInclusive"];
 pub const RANGE_TO_STD: [&str; 3] = ["std", "ops", "RangeTo"];
 pub const RC: [&str; 3] = ["alloc", "rc", "Rc"];
+pub const RECIEVER: [&str; 4] = ["std", "sync", "mpsc", "Receiver"];
```

Comment:
```suggestion
pub const RECEIVER: [&str; 4] = ["std", "sync", "mpsc", "Receiver"];
```

Reply:
Thank you for the suggestion. Fixed it in last commit